fvtest/porttest/si.cpp

```diff
@@ -2343,3 +2343,37 @@ TEST(PortSysinfoTest, sysinfo_cgroup_get_memlimit)
 	reportTestExit(OMRPORTLIB, testName);
 	return;
 }
+
+/**
+ * Test omrsysinfo_get_processor_description.
+ */
+TEST(PortSysinfoTest, sysinfo_test_sysinfo_get_processor_description)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	const char *testName = "omrsysinfo_get_processor_description";
+	int32_t rc = 0;
+	int i = 0;
+
+	OMRProcessorDesc desc;
+
+	rc = omrsysinfo_get_processor_description(&desc);
+	ASSERT_FALSE(rc == -1);
+
+#if (defined(J9X86) || defined(J9HAMMER))
+	ASSERT_FALSE(desc.processor < PROCESSOR_X86_UNKNOWN);
```

Comment:
This test could be:
```C++
ASSERT_GE(desc.processor, PROCESSOR_X86_UNKNOWN);
```
This way, In the case of a test failure, gtest will be able to print what the two compared values are.